src/ElectricPokemon.java

```diff
@@ -0,0 +1,36 @@
+public class ElectricPokemon extends PokemonBase {
+
+    private int voltage;
+    private int amperage;
+
+    public ElectricPokemon(String name, int voltage, int amperage){
+        super(name, Type.ELECTRIC);
+        this.voltage = voltage;
+        this.amperage = amperage;
+    }
+
+    public void voltTackle() {
+        System.out.println(getName() + " Volt tackle!");
```

Comment:
Het is prima dat je hier `getName()` gebruikt, maar weet dat je ook gewoon `name` of `this.name` kunt gebruiken